src/styles/index.scss

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__hands {
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+    display: flex;
+    position: relative;
+
+    &-min {
+      transform-origin: 0;
+      background-color: #0700ff;
+      width: 20vmin;
+      height: 3vmin;
+      animation-name: spin-20vmin-3600s;
+      animation-duration: 3600s;
+      animation-timing-function: steps(60, end);
+      animation-iteration-count: infinite;
+      position: absolute;
+      transform: translate(10vmin) rotate(-90deg);
+    }
+
+    &-sec {
+      transform-origin: 0;
+      background-color: #2c8000;
+      width: 38vmin;
+      height: 1.5vmin;
+      animation-name: spin-38vmin-60s;
+      animation-duration: 60s;
+      animation-timing-function: linear;
+      animation-iteration-count: infinite;
+      position: absolute;
+      transform: translate(19vmin) rotate(-90deg);
+    }
+
+    &-center {
+      background-color: #f6a603;
+      border-radius: 50%;
+      width: 5vmin;
+      height: 5vmin;
+      position: absolute;
+    }
+  }
+
+  &--speed-up &__hands-sec {
+    animation-duration: 10s;
+  }
+
+  &--speed-up &__hands-min {
+    animation-duration: 600s;
+  }
+}
+
+@keyframes spin-20vmin-3600s {
+  0% {
+    transform: translate(10vmin) rotate(-90deg);
+  }
+
+  100% {
+    transform: translate(10vmin) rotate(270deg);
+  }
+}
+
+@keyframes spin-38vmin-60s {
+  0% {
+    transform: translate(19vmin) rotate(-90deg);
+  }
+
+  100% {
+    transform: translate(19vmin) rotate(270deg);
+  }
 }
```

Comment:
```suggestion

```
Consider using one keyframe for both animations

Reply:
Edited my code.